presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -144,12 +144,49 @@ If an error similar to the following example is displayed, this is probably beca
     	at org.projectnessie.client.http.impl.jdk8.UrlConnectionRequest.executeRequest(UrlConnectionRequest.java:71)
     	... 42 more
 
+REST catalog
+^^^^^^^^^^^^
+
+To use a REST catalog, configure the catalog type as
+``iceberg.catalog.type=rest``. A minimal configuration includes:
+
+.. code-block:: none
+
+    connector.name=iceberg
+    iceberg.catalog.type=rest
+    iceberg.rest.uri=https://localhost:8181
+
+Additional supported properties for the REST catalog:
+
+==================================================== ============================================================
+Property Name                                        Description
+==================================================== ============================================================
+``iceberg.rest.uri``                                 REST API endpoint URI (required).
+                                                     Example: ``https://localhost:8181``
+
+``iceberg.rest.auth.type``                           The authentication type to use.
+                                                     Available values are ``NONE`` or ``OAUTH2`` (default: ``NONE``).
+                                                     ``OAUTH2`` requires either a credential or token.
+
+``iceberg.rest.auth.oauth2.credential``              The credential to use for OAUTH2 authentication.
+                                                     Example: ``key:secret``
+
+``iceberg.rest.auth.oauth2.token``                   The Bearer token to use for OAUTH2 authentication.
+                                                     Example: ``SXVLUXUhIExFQ0tFUiEK``
+
+``iceberg.rest.session``                             The session type to use when communicating with the REST catalog.
```

Comment:
I need to change this to `iceberg.rest.session.type` to match the config - noting it here as a reminder to myself